ra/next/ra-api.ng

```diff
@@ -76,12 +76,22 @@
 					<value>reload</value>
 					<value>meta-data</value>
 					<value>verify-all</value>
+					<!-- master/slave actions -->
+					<value>promote</value>
+					<value>demote</value>
+					<value>notify</value>
 				</choice>
 			</attribute>
 			<attribute name="timeout" />
 			<optional><attribute name="interval" /></optional>
 			<optional><attribute name="start-delay" /></optional>
 			<optional><attribute name="depth" /></optional>
+			<optional><attribute name="role">
+				<choice>
+					<value>master</value>
+					<value>slave</value>
+				</choice>
+			</attribute></optional>
```

Comment:
If we're codifying existing usage for 1.1, we need to stick with Pacemaker's capitalization; if we're planning new syntax for 1.2, I think we should go with "promoted" and "unpromoted".

Reply:
ok